docs/source/cluster_management/clis.rst

```diff
@@ -29,6 +29,18 @@ Right now, the best way to get the CLI is by cloning the atlasdb source code and
 
 This will create a zip file in ``atlasdb-cli/build/distributions``, which you can unzip on the machine where your AtlasDB server is installed.
 
+Offline CLIs
+============
+
+Due to their destructive nature for in-progress transactions, there are some CLIs which can must be run when AtlasDB is offline. These are:
+
+  - ``clean-transactions``
+  - ``fast-forward``
+
+To run these CLIs, first ensure that all of your AtlasDB clients are all shut down, and then run the CLI with the ``--offline`` flag, for example ``./bin/atlasdb --offline timestamp fast-forward``.
```

Comment:
We should probably add the flag needed for specifying a configuration to this example.